lib/prefab/options.rb

```diff
@@ -111,6 +113,7 @@ module DATASOURCES
 
       @sse_sources = @sources
       @config_sources = @sources
+      @migration_fallback = Prefab::MigrationFallback.new(migration_fallback)
```

Comment:
I think we want to make this assignment conditional based on the presence of `migration_fallback`